CHANGELOG.md

```diff
@@ -34,8 +37,57 @@
 
 ## Internal changes
 
+
+# [2021-08-02]
+
+## Release Notes
+
+If you want to set the default for file sharing in all teams to `disabled`, search for "File Sharing" in https://github.com/wireapp/wire-server/tree/develop/docs/reference/config-options.md.
+
+## Release Notes for Wire.com Cloud operators
+
+Upgrade nginz (#1658)
+
+## API Changes
+
+## Features
+
+* A new team feature for classified domains is available (#1626):
+  - a public endpoint is at `GET /teams/:tid/features/classifiedDomains`
+  - an internal endpoint is at `GET /i/teams/:tid/features/classifiedDomains`
```

Comment:
Should this have been mentioned in the API Changes section too?

Reply:
you're right, that would have been better.  in my defense, it's an extension, not a change in existing end-points, and the changelog entry isn't too long to read it end-to-end, and it's already release, so i won't fix it this time.